vnext/Shared/OInstance.cpp

```diff
@@ -412,6 +406,7 @@ InstanceImpl::InstanceImpl(
           [[fallthrough]];
 #endif
         }
+        case JSIEngineOverride::Chakra:
         case JSIEngineOverride::ChakraCore:
```

Comment:
>ChakraCore [](http://example.com/codeflow?start=32&length=10)

Should we remove ChakraCore from this enum then?

Reply:
We sure can.
We only have to make sure we also remove it from the Office-side equivalent when applying this update on its code base.

Reply:
Removing it from OSS should be fine... I think the only one to ever use ChakraCore UWP was "Mercury" (follow up offline for more details), which has moved to Hermes since.